test/hotspot/jtreg/ProblemList.txt

```diff
@@ -116,6 +116,7 @@ applications/jcstress/copy.java 8229852 linux-all
 containers/docker/TestJcmd.java 8278102 linux-all
 containers/docker/TestMemoryAwareness.java 8303470 linux-all
 containers/docker/TestJFREvents.java 8327723 linux-x64
+containers/systemd/SystemdMemoryAwarenessTest.java 8322420 linux-all
```

Comment:
This line should be removed as long as it gets applied after [17198](https://github.com/openjdk/jdk/pull/17198).

Reply:
Sure. We need to see which one goes in first and I'll adjust accordingly.